tests/mockserver.py

```diff
@@ -190,17 +197,37 @@ def render(self, request):
 
 class MockServer():
 
+    HTTP = 'mock_http'
+    HTTPS = 'mock_https'
+
+    @classmethod
+    def from_mock(cls, path, is_secure=False):
```

Comment:
What do you think about renaming it to `get_url`?

Reply:
Not a problem for me